filters/imagefilter.go

```diff
@@ -53,6 +53,15 @@ func HandleImageResponse(ctx filters.FilterContext, f ImageFilter) {
 }
 
 func handleImageTransform(out *io.PipeWriter, in io.ReadCloser, f ImageFilter) error {
+	var err error
+
```

Comment:
@niebes I don't remember... you can take some inspiration from [skipper](https://github.com/zalando/skipper/blob/8fb107612ac6543f1c76d01bb3e8efe61dac5beb/filters/builtin/compress.go#L310).